woof-code/rootfs-skeleton/usr/bin/xsaneshell

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 #8.06.11 rodin.s: i18n
 
+xsane -v >/dev/null 2>/dev/null
+if [ $? -eq 127 ]; then
```

Comment:
`if [ $? -gt 0]; then`

Reply:
Why need to check it for greater than 0 when the shell will return error code 127 only when program is not found? Note that if the program was found but in case it returned 1 or some other error code, this script to test whether the program existed will fail. So why not just check if it returned 127?